docs: restructure and add more content (#2232)
caos/zitadel: docs/docs/concepts/policies.md

## General

You can find these settings in the menu organization in the section polcies.

> Each policy can be overridden and resetted to the default.

Reviewer: ```suggestion
Each policy can be overridden and reset to the default.
```

Create a new identity provider configuration and enable it in the list afterwards.

For a detailed guide about how to configure a new identity provider for identity brokering have a look at our guide:
[Identity Brokering](../guides/authentication/identity-brokering)

## Lockout Policy

> Define when a user should be locked.

Reviewer: ```suggestion
Define when an account should be locked.
```

> The following properties are possible:

Reviewer: ```suggestion
The following settings are available:
```

> - Maximum Password Attempts: When the user has reached the maximum password attempts the user will be locked

Reviewer: ```suggestion
- Maximum Password Attempts: When the user has reached the maximum password attempts the account will be locked
```

> If a user is locked, an administrator has to unlock it in the ZITADEL console

Reviewer: ```suggestion
If an account is locked, the administrator has to unlock it in the ZITADEL console
```
